internal/querynode/task.go

```diff
@@ -793,6 +795,9 @@ func (r *releaseCollectionTask) Execute(ctx context.Context) error {
 	}
 	r.node.historical.removeGlobalSegmentIDsByCollectionID(r.req.CollectionID)
 
+	runtime.GC()
```

Comment:
debug.FreeOSMemory seems to trigger runtime.GC automatically

Reply:
```golang
//go:linkname runtime_debug_freeOSMemory runtime/debug.freeOSMemory
func runtime_debug_freeOSMemory() {
	GC()
	systemstack(func() { mheap_.scavengeAll() })
}
```

you're right

Reply:
removed